stakater-unit-test/helm/templates/clustertask.yaml

```diff
@@ -9,6 +9,25 @@ spec:
       script: |
         mvn clean test
       workingDir: $(workspaces.source.path)
+      when:
+        - input: "$(workspaces.source.path)/*.xml"
```

Comment:
@OsamaKhan220 there can be many types of xml files; so, we should look for specific xml file